assemblyline/odm/models/tagging.py

```diff
@@ -1,5 +1,6 @@
 from assemblyline import odm
 
+JA4_REGEX = r"(t|q)([sd]|[0-3]){2}(d|i)\d{2}\d{2}\w{2}_[a-f0-9]{12}_[a-f0-9]{12}"
```

Comment:
This should be here: https://github.com/CybercentreCanada/assemblyline-base/blob/0d1290f92fb0cc9bccdfa190a13640b570df33a3/assemblyline/odm/base.py#L89